content/overview/overview/building-process.md

```diff
@@ -11,15 +11,16 @@ aliases:
 
 ## How long it took to build
 
-After exploring configuration-management based approaches and tools, we turned instead to the Platform as a Service (PaaS) model. By building on Cloud Foundry, we were able to deploy a functioning PaaS quickly, then fine-tune it with each project we launched on it. The first prototype of the platform took about three months to complete, and production apps started launching about a month after that.
+After exploring configuration-management based approaches and tools, engineers at 18F turned instead to the platform-as-a-service (PaaS) model. By building on Cloud Foundry, the cloud.gov team was able to deploy a functioning PaaS quickly, then fine-tune it with each project an 18F product team launched on it. The first prototype of the platform took about three months to complete, and teams began to deploy their work to production about a month after that.
+launching about a month after that.
 
-As an agile project, cloud.gov is always under development. As we observe its use, we will continue to add new features that move the platform forward. Therefore, we do not refer to it as “complete” and probably never will.
+As an agile project, cloud.gov is always under development. The platform will continue to grow based on observations of how it is used, and based on direct feedback we receive from users.
```

Comment:
My 2 cents:

I don't have issue with the language, though I do wonder if we're always under development bc we're agile . . . I think that we could be continuously making improvements regardless of the management methodologies that we use. Also, perhaps a more meaningful thing to communicate is less about our agile practices (that's about us, really) and more about our commitment to human centered design (and are relying on feedback from our users to steer our work). I think the benefit of talking about HCD, rather than agile, is that it invites our users to interact with us, whereas mentioning agile doesn't really do that so much - particularly for readers who don't know much about it.
Thoughts @jameshupp?